tests/infrastructure/test_data_manager_stack.py

```diff
@@ -8,7 +8,7 @@ def test_sds_data_manager_stack(app, sds_id):
     stack = SdsDataManagerStack(app, stack_name, sds_id)
     template = Template.from_stack(stack)
 
-    template.resource_count_is("AWS::S3::Bucket", 1)
+    template.resource_count_is("AWS::S3::Bucket", 2)
```

Comment:
Can you add another `has_resource_properties()` test below confirming the bucket name of your new bucket?

Reply:
Added another has_resource_properties test for the new bucket